File: тесты/fib_recursive.py
import time
import sys

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

if __name__ == "__main__":
    n = 25
    if len(sys.argv) > 1:
        n = int(sys.argv[1])
    
    t0 = time.perf_counter()
    ans = fib(n)
    t1 = time.perf_counter()
    ms = (t1 - t0) * 1000.0
    print(f"RESULT:{ans}")
    print(f"TIME_MS:{ms:.2f}")

File: тесты/fib_recursive.cpp
#include <iostream>
#include <chrono>
#include <cstdlib>

long long fib(long long n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(int argc, char* argv[]) {
    long long n = 25;
    if (argc > 1) {
        n = std::atoll(argv[1]);
    }
    auto t0 = std::chrono::high_resolution_clock::now();
    long long ans = fib(n);
    auto t1 = std::chrono::high_resolution_clock::now();
    double ms = std::chrono::duration<double, std::milli>(t1 - t0).count();
    std::cout << "RESULT:" << ans << std::endl;
    std::cout << "TIME_MS:" << ms << std::endl;
    return 0;
}
